components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationConstants.java

```diff
@@ -214,6 +214,19 @@ private ApplicationInboundTableColumns() {
         }
     }
 
+    /**
+     * Grouping of constants related to database SP_SHARED_APP table.
+     */
+    public static class ApplicationSPSharedTableColumns {
+
+        public static final String SHARED_IDP_ID = "SHARED_IDP_ID";
+        public static final String OWNER_ORG_ID = "OWNER_ORG_ID";
+
+        private ApplicationSPSharedTableColumns() {
+
+        }
```

Comment:
since SP_SHARED_APP table related queries are not managed in framework, it is not recommended to redefine column names of SP_SHARED_APP here.
Hope you can't even use import the constants from other repo due to cyclic dependencies.


Can't we take an approach like this. 
* Implement another manager of https://github.com/wso2/carbon-identity-framework/blob/master/components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/DiscoverableApplicationManager.java (name suggestion: sharedAppDiscoverableManager) in [organizationApplication](https://github.com/wso2-extensions/identity-organization-management/tree/main/components/org.wso2.carbon.identity.organization.management.application) 
* From the places where it invokes DiscoverableApplicationManager methods, pick the correct implementation. Evaluate whether we can do this without any cyclic dependency

Reply:
Couldn't we define the constants in the organization-mgt-core repo and use that instead of moving the logic to the organization management repo?. Was this comment or any other concern lead to move the logic. @asha15

Reply:
I restructured the code in a way we don't need this constant anymore. However having SP_SHARED_APP related queries in framework repo will give maintainability issues. that's why we moved it to organization management repo.